src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let res = sourceString.split(';');
```

Comment:
Consider adding a check to handle cases where `sourceString` might be empty or not a string. This can prevent potential runtime errors.